Point_set_processing_3/include/CGAL/structure_point_set.h

```diff
@@ -1552,19 +1551,20 @@ class Point_set_with_structure
 template <typename PointRange,
           typename PlaneRange,
           typename OutputIterator,
-          typename NamedParameters
+          typename NamedParameters = parameters::Default_named_parameters
```

Comment:
You have just documented that the plane index map is mandatory, so why is there a default NP value in that function?

Reply:
That's something I indeed discussed with Laurent but I realize it did not worth handling the backward compatibility issue.